### PR TITLE
[CORE] Add new permissions asedit.placement and asedit.gravity & add pt_BR translation

### PR DESCRIPTION
**Pull request by thiagogebrimm**
#### Description:
New Permission Nodes asedit.placement and asedit.gravity
add pt_BR translation

----
### [CORE] Changes
*Changes to the core of the plugin - Performance Fixes, Bug Fixes, New Features, New Permission Nodes, New Config Options etc.* 

- [x] [CORE] New Permissions - asedit.placement & asedit.gravity
- [x] [CORE] New Translation - pt_BR 

----
### [CI] Changes
*Changes relating to the Continuous Integration of other Plugin APIs, Github Workflows, Issue Templates etc.*

----
### [DOC] Changes
*Changes relating to plugin Documentation - See the Wiki for more info*

----
### [MISC] Changes
*Changes that does not fit in the above list*

____
- [x] I have tested this pull request for defects on a server.

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the ArmorStandEditor Project Owners have the copyright to use and modify my contribution under the ArmorStandEditor [License](https://github.com/Wolfieheart/ArmorStandEditor/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

**Comment by Wolfieheart** [comment]
Looks good to me. Add the permissions as well to Plugin.yml and I can set this to Merge in :)

**Comment by Wolfieheart** [comment]
Also one final note: Rebase this into the next development version - dev/1.19.x-36 branch 👍 

Just a small note since that this the currently being worked on version and should allow for the build to work as expected.

File: src/main/java/io/github/rypofalem/armorstandeditor/menu/Menu.java [hunk 1]
```diff
@@ -152,18 +152,22 @@ private void fillInventory() {
...
+        if (pe.getPlayer().hasPermission("asedit.gravity")) {
+            gravity = createIcon(new ItemStack(Material.SAND), "gravity", "mode gravity");
+        }
 
         plate = createIcon(new ItemStack(Material.STONE_SLAB, 1),
                 "baseplate", "mode baseplate");
 
-        place = createIcon(new ItemStack(Material.MINECART, 1),
-                "placement", "mode placement");
+        if (pe.getPlayer().hasPermission("asedit.placement")) {
```
Comment (Wolfieheart) [inline src/main/java/io/github/rypofalem/armorstandeditor/menu/Menu.java:162]: Add this into resources/plugin.yml and add it to the Basic Perm Pack and Admin Perm Pack as required, please :)

Set it true like with asedit.basic - I consider all of this as part of the asedit.basic permission at the end of the day, but still a nice to have config option.

File: src/main/java/io/github/rypofalem/armorstandeditor/ArmorStandEditorPlugin.java [hunk 2]
```diff
@@ -112,7 +111,10 @@ public ArmorStandEditorPlugin(){
     @Override
     public void onEnable(){
 
-        scoreboard = this.getServer().getScoreboardManager().getMainScoreboard();
+        //Run UpdateChecker
+        runUpdateChecker();
+
+        scoreboard = Objects.requireNonNull(this.getServer().getScoreboardManager()).getMainScoreboard();
```
Comment (Wolfieheart) [inline src/main/java/io/github/rypofalem/armorstandeditor/ArmorStandEditorPlugin.java:117]: Nicely caught :)

File: src/main/java/io/github/rypofalem/armorstandeditor/ArmorStandEditorPlugin.java [hunk 3]
```diff
@@ -414,10 +419,9 @@ public boolean isEditTool(ItemStack itemStk){
             //If the ItemStack does not have Metadata then we return false
             if(!itemStk.hasItemMeta()) { return false; }
 
-            Integer itemCustomModel = itemStk.getItemMeta().getCustomModelData();
-            if (itemCustomModel == null) { return false; }
+            Integer itemCustomModel = Objects.requireNonNull(itemStk.getItemMeta()).getCustomModelData();
 
-            if(!itemCustomModel.equals(customModelDataInt)) { return false;	}
+            return itemCustomModel.equals(customModelDataInt);
```
Comment (Wolfieheart) [inline src/main/java/io/github/rypofalem/armorstandeditor/ArmorStandEditorPlugin.java:424]: Nicely caught :)

**Review by Wolfieheart — CHANGES_REQUESTED** [review]
Few Minor Changes mostly in having permissions added in to plugin.yml as requested. 
Comments like Nicely Caught :) Do not require change and are already marked as resolved. 
https://github.com/Wolfieheart/ArmorStandEditor/pull/96#issuecomment-1179676644 as well for good measure :) 

Good Job!

**Comment by thiagogebrimm** [comment]
I changed the plugin.yml with the new permissions

**Comment by Wolfieheart** [comment]
Can you also rebase dev/1.19.x-36 onto that Branch as this would be going into Dev Build First before the Main ASE Branch :)

**Comment by thiagogebrimm** [comment]
[v](https://github.com/Wolfieheart/ArmorStandEditor/pull/97) rebased

**Pull request closed without merging**